bundle/edu.gemini.ags/src/main/scala/edu/gemini/ags/gems/GemsMagnitudeTable.scala

```diff
@@ -102,12 +102,19 @@ object GemsMagnitudeTable extends MagnitudeTable {
     def getNominalMagnitudeConstraints(cwfs: CanopusWfs): MagnitudeConstraints
   }
 
+  // Values provided by science are at:
+  //      SB Any, CC 70, IQ 70.
+  // Add 0.8 to faintness / brightness limits to get values for the calculator, which assumes:
+  //      SB  20, CC 50, IQ 70.
```

Comment:
I can't remember any of this stuff but this really doesn't sound familiar.  Could you point out where this difference is coming from to refresh my memory?

Reply:
I found this incredibly confusing to determine. I looked at the old mag limits here and compared them to what science gave us, and the ones here were fainter by 0.8 mag. It took some mucking around in the `edu.gemini.catalog` api to figure out how to explain the difference.

Now that I'm looking again, this may not even be the correct explanation, as both CC70 and SB20 could explain the 0.3 difference, and SB Any or IQ85 could explain the 0.5 difference.

In any case, I think I should eliminate this comment as I'm not convinced of it now, and just note that these are 0.8 mag different than the values typically supplied by science.

Reply:
i vaguely remember a google excel sheet where those numbers were stored. Do you have access to that?

Reply:
There is a CSV file where some of the values are stored, but it doesn't include the values for any GeMS guide probes.

Reply:
See here:
https://github.com/gemini-hlsw/ocs/blob/develop/bundle/edu.gemini.ags/src/main/resources/edu/gemini/ags/conf/Guide%20Limits%20-%20OT%20Config.csv

Reply:
Also, from the top of this file, the comment:

"A magnitude table defined in the same way as we have done since before 2015A (that is, with predefined magnitude limits)."

I can't exactly remember how this worked either as it's been a couple years, but the values here required different handling than other probe limits.

Reply:
So, for the other guide probes we are explicitly provided values for all the SB, and IQ combinations, and make an adjustment for CC.

Reply:
@swalker2m Yes, exactly. For GeMS, we only get one set of mags, which we adjust using the code in:
https://github.com/gemini-hlsw/ocs/blob/develop/bundle/edu.gemini.catalog/src/main/scala/edu/gemini/catalog/api/package.scala

A bit confusing, really, especially given that science did not provide the conditions for the mag limits in the NGS2 OT doc.

Reply:
Let's get feedback on what the assumed conditions are for the new limits, but we can merge this in the meantime and come back to it if we need to make a change. 👍

Reply:
So it turns out that the values provided by science are SB ANY, CC50, IQ70.
Thus, I think the mag limits for Canopus in previous versions of the OT are wrong and should have been -0.3 mag what they actually are.

We must add +0.5 mag to the limits given by science in order for them to account for SB ANY, which results in a -0.5 mag during condition adjustment.

Reply:
Are those new values for NGS2?

Reply:
Whoops. I didn't see this before I merged. Sorry, @cquiroz.

Yes, those are the new NGS2 values as confirmed by science for the conditions in the comment.

Reply:
Excellent, I was worried they were the current valid ones and we have had them wrong in the OT forever